cmd/certificates/auth.go

```diff
@@ -49,6 +50,23 @@ var (
 )
 
 func init() {
+	// NB: init functions are executed in lexicographical order of filename
+	confDirParam := cfgstruct.FindConfigDirParam()
```

Comment:
Can we do all of the setup in main instead, that way it's clearer for order.

Reply:
this stuff has to be here otherwise it won't be defined in time for other `init` functions

Reply:
I mean if we remove all init functions and do everything in `func main()` then there shouldn't be problems.

Reply:
ah - misunderstood... sure!